Runtime/AvatarCreator/UI/PhotoCaptureElement.cs

```diff
@@ -64,6 +85,35 @@ public void TakePhoto()
             onPhotoCaptured?.Invoke(texture);
         }
 
+        private async Task GetPermission()
```

Comment:
more of a question but if they aren't targeting mobile then this function essentially does nothing? 

Is there some sort of permission API for Standalone/desktop Windows and Mac? 🤔

Reply:
I didn't found any such settings in build setting or api for unity

Reply:
cool just wondering :D